src/Pages/Catagories.js

```diff
@@ -0,0 +1,12 @@
+import NavBar from '../Components/NavBar';
+
+const Catagories = () => (
+  <>
+    <NavBar />
+    <div className="categories-container">
+      <p>The initial page!</p>
+    </div>
```

Comment:
The `Categories` component should display only a button with the text "Check status" as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/190614550-bec6f612-7d37-47ef-b6ec-98abc02110d3.png)